### PR TITLE
Add validation script: Validation email field on client-side

### PR DESCRIPTION
**Pull request by JbirdL86**
In this feature we've added a validation function to the form element, it checks if the email address is in lower case and shows an error if it's capitalized. The main files modified during the creation of this feature are:

- index.html
- validation.js

I hope you like my project so far :+1:

### CODE REVIEW

File: validate.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
+const submitButton = document.getElementById('submit');
+const email = document.getElementById('mail');
+
+email.value = 'FErnando@gmail.com';
```
Comment (teekaytech) [inline validate.js]: I think you do not need to give a default email address. You should only validate based on whatever input available on the `email` field at any point in time when the submit button is clicked.

**Review by teekaytech — CHANGES_REQUESTED** [review]
## Status: Changes Required :warning: :vertical_traffic_light: 

### Hello @JbirdL86 :wave:

_You've really done an amazing job so far. In this PR, here are my major highlights_

- Your linter checks are running with no errors :100: 
- You've prepared a professional PR with descriptive title and summary :+1: 
- The validation feature simply works :clap: 

__However, you need to make some required changes before moving on to the next assignment.__
> Keep in mind that your long hours of work are appreciated. However, the code review process is for improving the code you wrote, and not for criticizing it :blush: 

### **Required Changes** :recycle: 
Check the comments under the review.

### **Optional Changes** :zap: 
N/A


* * *
**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear. You can mention me (@teekaytech) so I can be notified about your comments and respond._

*REMINDER: As described in the [Code reviews limit Policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).*

_**Please make the required changes and submit another review request when you are done.**_

* * *

_Reviewed by:_ [**Taofeek Olalere**](https://github.com/teekaytech)
__Cheers & Happy coding :smiley:__

File: validate.js [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+const email = document.getElementById('mail');
+
+submitButton.addEventListener('click', () => {
+  const emailString = email.value;
+
+  if (emailString === emailString.toLowerCase()) {
+    submitButton.setCustomValidity('');
+  } else {
+    submitButton.setCustomValidity('Email input characters must be lower case!');
+    submitButton.preventDefault();
```
Comment (MahmoudBakr23) [inline validate.js:11]: - [ ] **OPTIONAL**
According to the project requirements, you should dynamically render an error message that appears next to the get in touch button and style it as the color / pattern of the website.

_I'm sorry this point didn't show up in the earlier review._

**Review by MahmoudBakr23 — APPROVED** [review]
Hi @JbirdL86 😃, 

Well done on the project and on the requested changes. However, there is always room for improvement. Kindly check the comment below. 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit:
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**